src/interfaces/stepper-interfaces/YourCompany/address-interface.ts

```diff
@@ -1,31 +1,31 @@
 /** Interface to define a base address. */
 export interface AddressIF {
-  addressCity: string;
-  addressCountry: string;
-  addressRegion: string;
-  deliveryInstructions?: string;
-  postalCode: string;
-  streetAddress: string;
-  streetAddressAdditional?: string;
+  addressCity: string
+  addressCountry: string
+  addressRegion: string
+  deliveryInstructions?: string
+  postalCode: string
+  streetAddress: string
+  streetAddressAdditional?: string
```

Comment:
All this cleanup isn't frivolous. I was originally going to put the "editing" flags in their respective step interface and I cleaned up along the way. But things were too inconsistent so I did it differently. And I thought I'd keep the cleanup work I did.